log.go

```diff
@@ -32,83 +32,75 @@ const (
 	LevelDebug
 )
 
-// BeeLogger references the used application logger.
-var BeeLogger = logs.NewLogger(100)
```

Comment:
I think we still need BeeLogger. As there's so many application rely on this variable.